README.md

```diff
@@ -52,6 +52,26 @@ $ docker run \
   -p "4000:4000" starefossen/github-pages
 ```
 
+## Docker compose
+
+If you want to use Docker Compose instead, you can move a lot of the options into a `docker-compose.yml` at the root of your project:
+
+```
+version: '3'
+services:
+  jekyll:
+    image: starefossen/github-pages
+    environment:
+      - "JEKYLL_GITHUB_TOKEN:${JEKYLL_GITHUB_TOKEN}"
+    ports:
+      - "4000"
+    volumes:
+      - ./:/usr/src/app
+    tty: true
+```
+
+Then start the container with `docker-compose run --rm --service-ports jekyll`.
```

Comment:
This can be simplified to `docker-compose up`.

Reply:
You're right. I think the "--service-ports" flag and my config above had something to do with eachother, but now I can't get them to work anymore. Maybe a different version of the docker engine?

Reply:
```suggestion
Then start the container with `docker-compose up`.
```